mobile-app/src/components/PlayableCell/playableCellService.ts

```diff
@@ -1,5 +1,5 @@
-import type { Cell } from '../../../web-app/src/common/gameState';
-import { DEFAULT_CELL_COLOR, PLAYER_1_CELL_COLOR, PLAYER_2_CELL_COLOR } from "../../utils";
+import type { Cell } from '../../../../web-app/src/common/gameState';

```

Comment:
Do not use code outside of your project, prefer duplicate this file instead. You can have problem during the app's compilation.